pyproject.toml

```diff
@@ -152,8 +152,8 @@ portalocker = "^2.7.0"
 structlog = "^23.1.0"
 structlog-sentry = "^2.0.2"
 tensorflow-io-gcs-filesystem = [
-    {version = ">=0.23.1,<0.32", markers = "sys_platform == 'win32'"},
-    {version = ">=0.32", markers = "sys_platform != 'win32'"}
+    {version = "==0.31", markers = "sys_platform == 'win32'"},
+    {version = "==0.32", markers = "sys_platform != 'win32' and sys_platform != 'darwin'"}
```

Comment:
probably need to make the right distinction so that darwin x86 gets it but arm darwin does not